### ISSUE TITLE
Just updated to last version and get an error in Configuration Check

### ISSUE THREAD

**Issue by ciairmi** [custom-components/ble_monitor#552]
Hello. 
I've Just updated the integration from Hacs and I get this error

Invalid config for [ble_monitor]: value must be one of ['ATC', 'BlueMaestro', 'Brifit', 'Govee', 'Kegtron', 'Mi Scale', 'Moat', 'Off', 'Other', 'Qingping', 'Ruuvitag', 'SensorPush', 'Teltonika', 'Thermoplus', 'Xiaogui', 'Xiaomi', 'iNode'] for dictionary value @ data['ble_monitor']['report_unknown']. Got False. (See ?, line ?).

My HA runs on Rasp4 and Is updated to last 2021.11.2 version

Can you help me please?

![Screenshot_20211111-070547_Home Assistant](https://user-images.githubusercontent.com/75178980/141246815-1643575e-3097-4e4e-9d5a-71bfabc9ff13.jpg)

**Comment by Ernst79** [comment]
Are you sure you have the latest version? Did you update to 5.9.1? It should already be fixed, but you might need to force a restart of HA.

**Comment by ciairmi** [comment]
Yes I've updated to 5.9.1
I'm scared to force the reboot?
Are you sure?

**Comment by Ernst79** [comment]
The easiest is to change `report_unknown` to `Off`. This should make it pass the config validation.

**Comment by ciairmi** [comment]
Ok. How to make this change?

**Comment by Ernst79** [comment]
Go to settings, integrations and click on the config button of the BLE monitor tile

![image](https://user-images.githubusercontent.com/24375847/141276754-6d0e44f7-0d6a-4adc-9552-6d9d15ded471.png)

**Comment by ciairmi** [comment]
Hi Hernst,
sorry but I can't find it. look at the screen shots.
I've tried to insert in the text field _report_unknown=Off_ but it says that changes should be done directly in the yaml file, and I don't know how or where.

![1](https://user-images.githubusercontent.com/75178980/141317241-6196d3c8-9fd7-4c59-9f00-62a0a242d2b4.png)
![2](https://user-images.githubusercontent.com/75178980/141317246-1f32cc29-94d0-44a0-b6d4-e38c2a2853e4.png)
.

**Comment by ciairmi** [comment]
And If I click ok "Opzioni di sistema" i just get tha
![3](https://user-images.githubusercontent.com/75178980/141317824-52ed5ba0-117a-49e1-92f0-fdbe58986055.png)
t

**Comment by Ernst79** [comment]
Ah, sorry, you are apparently using YAML (I wrongly assumed you configured it in the UI). 

Look in your configuration.yaml file.

**Comment by ciairmi** [comment]
configuration.yaml is quite empty.

I just have a package that contains that:
device_tracker:
  - platform: bluetooth_tracker
  - platform: bluetooth_le_tracker

Is it the one's we need?

**Comment by Ernst79** [comment]
No, that's not the one. 

How did you configure BLE monitor than? You normally have 2 choices, in the UI or in YAML. HA is telling that you have used YAML to configure your component. So there should be something like 

```yaml
ble_monitor:
  bt_interface: '04:B1:38:2C:84:2B'
  discovery: True
  active_scan: False
  report_unknown: False <<<<<<<< CHANGE THIS LINE TO report_unknown: "Off"
  decimals: 1
  period: 60
  log_spikes: False
  use_median: False
  restore_state: False
```

**Comment by ciairmi** [comment]
Got It... I've used it into the Miflora package. BUT!!!!!!!
I've checked the configuration before rebooting, and it's "ok",
Unfortunately the reboot crashed and now HA is not available any more!
I've needed to flash the OS as a new installation and restore a backup!
Force the reboot wasn't a good idea.
Probably you ned to deeper understand why it happened!!!
Anyway thanks for you prompt support.

**Comment by Ernst79** [comment]
Sorry to hear that. 

The mi flora package is a different integration. Even if you have a wrong configuration in YAML, HA should just start normally, it should just not load the wrong integration. I doubt BLE monitor was the cause of the crash, to be honest. 

I would suggest that you use the UI configuration of BLE monitor in stead of YAML. Just add it in the integration menu and look for BLE monitor. You can than set all the configuration parameters in the UI. If you already have it in yaml right now, you can remove the configuration from Yaml. After a restart, you can use the UI configuration options.

**Comment by ciairmi** [comment]
Mi Flora is not strictly related to the BLE Monitor.
I've just inserted both in the same package.

The code I've found in the package is this:

ble_monitor:
  decimals: 1
  period: 90
  log_spikes: False
  use_median: true
  active_scan: False
  hci_interface: 0
  batt_entities: True
  discovery: True
  report_unknown: False     <<<<I'VE CHANGED THAT TO "OFF">>>>
  devices:
    - mac: 'C4:7C:8D:6C:45:B2'
      name: 'sala'
      temperature_unit: C

I'll try as you suggest and let you know.
Thanks ;o)
bye
